src/pages/index.js

```diff
@@ -30,7 +30,7 @@ const features = [
     ),
   },
   {
-    title: "APIs & Clients",
+    title: "APIs & Tools",
```

Comment:
@christinaausley I was going to make a comment here about how we realized we can't change the URL in this location yet, because it has the host hardcoded and the new `working-with-apis-tools` URL doesn't exist yet in production....but then I also noticed that this URL is for v8.1 anyway, and this PR doesn't include any v8.1 changes.

I'm not sure if that means you want to include the title change in this PR, or if you want to wait until all of this makes it into v8.1? I have no preference, but since you pointed out my misunderstanding of the versions impacted by this PR, I realized I was interpreting this specific change incorrectly all along.